scrapper.py

```diff
@@ -26,38 +36,133 @@ class Crawler:
     Crawler implementation
     """
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        class_bs = article_bs.find('div', class_='view-content view-rows')
+        title_bs = class_bs.find_all('td', class_="views-field views-field-title table__cell")
+        for link_bs in title_bs:
+            if len(self.urls) >= self.max_articles:
+                break
+            link_bs = link_bs.find('a')
+            self.urls.append('https://rjano.ruslang.ru' + link_bs['href'])
```

Comment:
this constant should be defined somwhere else, not in the function that concatenates URLs